src/org/intellij/grammar/KnownAttribute.java

```diff
@@ -52,6 +52,8 @@ public class KnownAttribute<T> {
   public static final KnownAttribute<String>       EXTENDS                   = create(false, String.class, "extends", BnfConstants.AST_WRAPPER_PSI_ELEMENT_CLASS);
   public static final KnownAttribute<ListValue>    IMPLEMENTS                = create(false, ListValue.class, "implements", ListValue.singleValue( null, BnfConstants.PSI_ELEMENT_CLASS));
   public static final KnownAttribute<String>       ELEMENT_TYPE              = create(false, String.class, "elementType", null);
+  public static final KnownAttribute<String> FALLBACK_STUB_ELEMENT_TYPE =
```

Comment:
do you mind keeping the formatting in the file? :)

Reply:
I did align the filed name